loader/windows/icd_windows_apppackage.cpp

```diff
@@ -19,6 +19,10 @@
 #include "icd.h"
 #include "icd_windows.h"
 
+extern "C" {
+    int khrEnableTrace;
+}
+
```

Comment:
Not sure about this one (don't know why it would be required), but it would seem more coherent with below conventions if it were written as:

```suggestion
extern "C" int khrEnableTrace;

```

Reply:
Leaving this open so @jenatali can take a look.

Reply:
Weird. No idea why it'd do that.

Alternatively you could use:
```c++
#ifdef __cplusplus
extern "C" {
#endif

...

#ifdef __cplusplus
}
#endif
```

In the `icd.h` header to avoid having to re-declare it here with the extern.